webapp/channels/src/components/add_user_to_group_multiselect/multiselect_option/multiselect_option.tsx

```diff
@@ -4,16 +4,16 @@
 import classNames from 'classnames';
 import React from 'react';
 
+import {BotTag} from '@mattermost/design-system';
 import type {UserProfile} from '@mattermost/types/users';
 import type {RelationOneToOne} from '@mattermost/types/utilities';
 
 import {Client4} from 'mattermost-redux/client';
 import {isGuest} from 'mattermost-redux/utils/user_utils';
 
+import GuestTag from 'components/guest_tag';
```

Comment:
I'm kind of confused by this. why is guest_tag in the components folder and a preset also in the design-system folder